src/styles/modules/_Resource.scss

```diff
@@ -1,9 +1,44 @@
 .resource-container {
```

Comment:
This card looks really good! Looking forward to getting a good-looking folder style to match